admin/index.html

```diff
@@ -12,7 +12,7 @@
 
   <body>
     <!-- Include the script that builds the page and powers Netlify CMS -->
-    <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
```

Comment:
really glad we're moving away from this CDN!

Reply:
👍

Reviewing some additional changes and dependencies. Will add another update.